tools/syft/go.mod

```diff
@@ -1,214 +1,272 @@
 module github.com/redhat-appstudio/tssc-dev-multi-ci/tools/syft
 
-go 1.23.6
+go 1.24.1
 
-require github.com/anchore/syft v1.19.0
+require github.com/anchore/syft v1.29.0
```

Comment:
@tnevrlka, @chmeliik do you see any obvious issues with making this update?

Reply:
Not that I can think of. The typical problem would be that syft starts defaulting to a version of CycloneDX that TPA can't handle, but the buildah-rhtap task pins the output version to 1.5. And the upload-sbom task has logic to downgrade to the supported version anyway. So that should be fine.